frontend-app/app/app.routes.js

```diff
@@ -21,24 +21,23 @@
               })
               .when('/profile', {
                   templateUrl: 'modules/profile/profile.view.html',
-                  controller: 'ProfileCtrl'
+                  controller: 'ProfileCtrl',
+                  requireAuth: true
+                  
                   /*resolve: {
                       loggedin: checkLoggedin
                   }*/
               }).when('/group-timeline', {
                  templateUrl: 'modules/group-timeline/group-timeline.view.html',
-                 controller: 'GroupTimelineCtrl'
+                 controller: 'GroupTimelineCtrl',
+                 requireAuth: true                 
              })
               
               .when('/group/:id', {
                  templateUrl: 'modules/group-timeline/group-timeline.view.html',
                  controller: 'GroupTimelineCtrl',
-                
+                 requireAuth: true                 
              })
-              .when('/user/:id', {
```

Comment:
The route is added for viewing other users' profile. Please add it again :smile: